cmd/generator/main_test.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"testing"
+)
+
+type testCase struct {
```

Comment:
let's specify what kind of `testCase` is (in the name).

It is useful if we will have another test in this file.

Reply:
What do you think about ```testSuccessfullGenerating``` as new struct name?